setup.py

```diff
@@ -5,15 +5,18 @@
 
```

Comment:
This pr relies on the added packages to properly run. I included the setup.py diff in the readme/infra pr, so when I committed this, I checked setup.py out from that branch so this would build.